py/picca/fitter2/xi.py

```diff
@@ -231,6 +231,30 @@ def qso_bias_vs_z_croom(z, tracer, zref = None, **kwargs):
     p1 = kwargs["croom_par1"]
     return (p0 + p1*(1.+z)**2)/(p0 + p1*(1+zref)**2)
 
+def sky_residual_correlation(r, mu, name, bin_size_rp, *pars, **kwargs):
+    '''Contribution to the correlation of delta field
```

Comment:
@londumas could you instead add an option to the current broadband scheme: e.g. `pre-sky` and call the function `sky_residual...` for this option? Also, could you rename `sky_residual_correlation` to `broadband_sky_res`?